projects/swimlane/ngx-charts/src/lib/common/tooltip/tooltip.directive.ts

```diff
@@ -20,7 +20,7 @@ import { TooltipService } from './tooltip.service';
 @Directive({ selector: '[ngx-tooltip]' })
 export class TooltipDirective implements OnDestroy {
   @Input() tooltipCssClass: string = '';
-  @Input() tooltipTitle: string = '';
+  @Input() tooltipTitle?: string;
```

Comment:
Needs to be `optional` since we are passing sometimes `undefined`.

See: https://github.com/swimlane/ngx-charts/pull/1647/files#diff-c371f94709506ceeaf97e00694ddbd2af8e52cf5e89ba4651a8af8ec2aed499fR48

`[tooltipTitle]="tooltipTemplate ? undefined : tooltipText(valueArc)"`